MapboxMobileEvents/MMEEventLogger.m

```diff
@@ -35,16 +51,18 @@ - (void)logEvent:(MMEEvent *)event {
 
 #pragma mark - Write to Local File
 
+- (BOOL)isTimeForNewLogFile {
```

Comment:
Another way of doing something like this is to make an internal property like `@property (nonatomic, readonly, getter=isTimeForNewLogFile) BOOL timeForNewLogFile`. The implementation you already have would implement the getter described in the property declaration.

Reply:
Thanks for pointing this out. I've added the internal property, it should help it to be a bit more readable and match the style of the project.